### PR TITLE
Add Description how to Start and Configure the Server

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+-it -p 8080:8080 livingdocs/release-server:latest
+```
+
+`RANCHER_URL` / `CATTLE_ACCESS_KEY` / `CATTLE_SECRET_KEY` are rancher specific variables to have access to your rancher via the rancher API.
+To configure a deployment you always have to pass 2 environment variables during startup.
+
+* `ld_mapping__<service1>__rancher_service_id=<service-id>`
+* `ld_mapping__<service1>__docker_image=<docker-image-name>`
+
+`<service1>` is a serviceHandle and is later used as an identifier to make deployment. Every rancher stack has stacks and every stack has services. If you click on a stack in the rancher gui, you will see the `service-id` of a service in the url. `docker-image-name` is the tag of a docker image pushed to dockerhub.
```
Comment (mthpvg) [inline README.md]: > Every rancher stack has stacks

I don't get what those meta stacks are. I understand that a stack has services and a service has containers.

> `docker-image-name` is the tag of a docker image

Could this be: `docker-image-name` is the **name** of a docker image

Comment (mthpvg) [inline README.md]: selection_086


Just to be sure the service-id is, in this screenshot: `1st615`?

**Review by mthpvg — APPROVED** [review]
Very nice thx.

File: README.md (hunk 1 above)
Comment (marcbachmann) [reply README.md]: > Every rancher stack has stacks and every stack has services.

That's a typo. Every rancher environment has stacks and every stack has a service.

Comment (marcbachmann) [reply README.md]: That's the stack id. The service id shows up in the url when you click onto a service.

**Review by marcbachmann — CHANGES_REQUESTED** [review]
Please fix the typo with the rancher environment.

File: README.md (hunk 1 above)
Comment (mthpvg) [reply README.md]: Thanks, got it!

Comment (mthpvg) [reply README.md]: Environment > stack > service > container :ok_hand:

File: README.md [hunk 2]
```diff
@@ -27,7 +27,8 @@ To configure a deployment you always have to pass 2 environment variables during
 * `ld_mapping__<service1>__rancher_service_id=<service-id>`
 * `ld_mapping__<service1>__docker_image=<docker-image-name>`
 
-`<service1>` is a serviceHandle and is later used as an identifier to make deployment. Every rancher stack has stacks and every stack has services. If you click on a stack in the rancher gui, you will see the `service-id` of a service in the url. `docker-image-name` is the tag of a docker image pushed to dockerhub.
+`<service1>` is a serviceHandle and is later used as an identifier to make a deployment. Every rancher environment has 1-n stacks, every stack has 1-n services and every service has 1-n containers. If you click on a service in the rancher gui, you will see the `service-id` of a service in the url f.e. `https://test.ch/env/<environment-id>/apps/stacks/<stack-id>/services/<service-id>/containers`. `docker-image-name` is the name of a docker image pushed to dockerhub.
```
Comment (marcbachmann) [inline README.md]: > Every rancher environment has 1-n stacks

That's wrong again. It has n stacks. There can be empty environments.

Comment (DaRaFF) [reply README.md]: Yes theoretically. But in real life it makes no sense to have n environment with 0 stacks.

Comment (marcbachmann) [reply README.md]: I'd just remove that instead of making a wrong statement.

Comment (DaRaFF) [reply README.md]: Boah today you are overprecise, but I will change it :-)

Comment (marcbachmann) [reply README.md]: Hehe. maybe sorry.

**Pull request merged**